eo-runtime/src/test/eo/org/eolang/nan-tests.eo

```diff
@@ -27,85 +27,122 @@
 +junit
 +version 0.0.0
 
-[] > nan-eq-is-valid
+[] > nan-not-eq-number
+  assert-that > @
+    nan
+    $.not
+      $.equal-to 42
+
+[] > nan-not-eq-nan
+  nan > not-a-number!
+  assert-that > @
+    not-a-number
+    $.not
+      $.equal-to not-a-number
+
+[] > nan-not-lt-number
+  assert-that > @
+    nan
+    $.not
+      $.less-than 42
+
+[] > nan-not-lt-nan
+  nan > not-a-number!
+  assert-that > @
+    not-a-number
+    $.not
+      $.less-than not-a-number
+
+[] > nan-not-lte-number
+  assert-that > @
+    nan.lte 42
+    $.equal-to FALSE
+
+[] > nan-not-lte-nan
+  nan > not-a-number!
+  assert-that > @
+    not-a-number.lte not-a-number
+    $.equal-to FALSE
+
+[] > nan-not-gt-number
+  assert-that > @
+    nan
+    $.not
+      $.greater-than 42
+
+[] > nan-not-gt-nan
   nan > not-a-number!
   assert-that > @
-    and.
-      (not-a-number.eq 42).eq FALSE
-      (not-a-number.eq not-a-number).eq FALSE
-    $.equal-to TRUE
+    not-a-number
+    $.not
+      $.greater-than not-a-number
+
+[] > nan-not-gte-number
+  assert-that > @
+    nan.gte 42
+    $.equal-to FALSE
 
-[] > nan-lt-is-valid
+[] > nan-not-gte-nan
   nan > not-a-number!
   assert-that > @
-    and.
-      (not-a-number.lt 42).eq FALSE
-      (not-a-number.lt not-a-number).eq FALSE
-    $.equal-to TRUE
+    not-a-number.gte not-a-number
```

Comment:
@maxonfjvipon We have similar code in several places:
```eo
not-a-number.gte not-a-number
  $.equal-to FALSE
```
Can we change it to something like:

```
$.not
  $.less-than not-a-number
``` 
or even:
```eo
$.greater-than-equals not-a-number
```
In order to avian using literals like `FALSE` or `TRUE` ?

Reply:
@volodya-lombrozo I did not found `greater-than-equals` in `eo-hamcrest` so it can't be used. Also here we test `nan.gte` object. If we do 

```
$.not
  $.less-than
```
`nan.gte` object will be never called

Reply:
@volodya-lombrozo WDYT?

Reply:
@maxonfjvipon I see. Then let's leave it as is.